spec/requests/api/v1/users_spec.rb

```diff
@@ -105,11 +98,81 @@
     context "when using a lead provider token" do
       let(:token) { LeadProviderApiToken.create_with_random_token!(lead_provider: create(:lead_provider)) }
 
-      it "returns 401 for invalid bearer token" do
+      it "returns 403 for invalid bearer token" do
         default_headers[:Authorization] = bearer_token
         get "/api/v1/users"
         expect(response.status).to eq 403
       end
     end
   end
+
+  describe "#create" do
+    context "when authorized" do
+      before do
+        default_headers[:Authorization] = bearer_token
+        default_headers["Content-Type"] = "application/vnd.api+json"
+      end
+
+      let(:json) do
+        {
+          data: {
+            type: "user", # is this singular or plural??? # spec is agnostic # i chose singular
```

Comment:
I would go for singular, and remove the comment,

Reply:
yep sorry, I chopped of that commit when i was removing the migrations